src/View/AdminUser/AddUserUI.java

```diff
@@ -0,0 +1,151 @@
+
+/**
+ * UI to Add User, Form with the fields of user
+ * @author: Lourdes Villca
+ */
+
+public class AddUserUI extends JPanel {
+    private JLabel nameLabel;
+    private JLabel lastNameLabel;
+    private JLabel userLabel;
+    private JLabel passwordLabel;
+    private JLabel activeLabel;
+    private JLabel addressLabel;
+    private JLabel emailLabel;
+    private JLabel roleLabel;
+    private JLabel possitionLabel;
+    private JTextField nameTxt;
+    private JTextField lastNameTxt;
+    private JTextField userTxt;
+    private JTextField passwordTxt;
+    private JCheckBox activeCbox;
+    private JTextField addressTxt;
+    private JTextField emailTxt;
+    private JComboBox roleCbox;
+    private JComboBox possitionCbox;
+    private JButton acceptBtn;
+    private JButton cancelBtn;
+    private UserEvent userEvent;
+    private JPanel leftPanel;
+    private JPanel rightPanel;
+
+    public AddUserUI(){
+
+        leftPanel = makeLeftPanel();
+        rightPanel = makeRightPanel();
+        acceptBtn = new JButton("Accept");
+        cancelBtn = new JButton("Cancel");
+        this.setLayout(new BorderLayout());
+
+        acceptBtn.addActionListener(new UserEvent(this));
+
+        JPanel bottomPanel = new JPanel();
+        bottomPanel.setLayout(new BoxLayout(bottomPanel, BoxLayout.X_AXIS));
+
+        bottomPanel.add(acceptBtn);
+        bottomPanel.add(cancelBtn);
+
+        this.add(leftPanel, BorderLayout.LINE_START);
+        this.add(rightPanel, BorderLayout.LINE_END);
+        this.add(bottomPanel, BorderLayout.PAGE_END);
+
+        this.setBorder(BorderFactory.createTitledBorder("Add New User"));
+    }
+    /**
+     * Method to build the left panel with the corresponding data
+     * @return return a panel with all their components
+     */
+    private JPanel makeLeftPanel(){
+        JPanel panel = new JPanel(new SpringLayout());
+        Dimension txtSize = new Dimension(150, 25);
+        InitComponent.initLabel(nameLabel, "Name", panel);
+        InitComponent.initTextBox(nameTxt, txtSize, panel);
+
+        InitComponent.initLabel(lastNameLabel, "Last Name", panel);
+        InitComponent.initTextBox(lastNameTxt, txtSize, panel);
+
+        InitComponent.initLabel(userLabel, "User", panel);
+        InitComponent.initTextBox(userTxt, txtSize, panel);
+
+        InitComponent.initLabel(passwordLabel, "Password", panel);
+        InitComponent.initTextBox(passwordTxt, txtSize, panel);
+        InitComponent.initLabel(activeLabel, "Active", panel);
+
+        activeCbox = new JCheckBox();
+        panel.add(activeCbox);
+
+        InitComponent.setSizeComponent(panel, new Dimension(310, 50));
+        SpringUtilities.makeCompactGrid(panel, 5, 2, 6, 6, 6, 6); /* rows, cols, initX, initY, xPad, yPad*/
+
+        return panel;
+    }
+    /**
+     * Method to build the right panel with the corresponding data
+     * @return return a panel with all their components
+     */
+    private JPanel makeRightPanel()
```

Comment:
instead of make try to use a buildRightPanel it is more accurate at what you are trying to express